training/COMPARISON.md

```diff
@@ -281,13 +280,14 @@ for i in array:
 NotImplementedError: pdarray does not support iteration. To force data transfer from server, use to_ndarray
 ```
 
-Unfortunately, there is a way to iterate `pdarray` objects without raising an exception. This is for informational purposes only. If you find yourself writing code similar to what appears below or wanting to iterate a `pdarray`, you will most likely need Chapel.
+Unfortunately, there is a way to iterate `pdarray` objects without raising an exception. This is for informational purposes only. If you find yourself writing code similar to what appears below or wanting to iterate a `pdarray`, here's likely a way to get the desired behavior with an array oriented method in arkouda or the functionality will need to be implemented in Chapel.
+
 ```python
 import arkouda as ak
 ak.connect()
 array = ak.arange(100)
 for i in array.to_list():
-    print(i)
+    print(array[i])
```

Comment:
if we are doing `array[i]` then we should be looping over `array` not `array.to_list`

Reply:
Oh right. I've gotten so used to doing it correctly. fixing it now

Reply:
Done

Reply:
note im dumb and messed this up the code block from the original comment was correct

```python
for i in range(array.size):
    print(array[i])
```